src/core/particles/webgl/ParticleRenderer.js

```diff
@@ -270,9 +269,9 @@ ParticleRenderer.prototype.generateBuffers = function ( container )
         this.properties[i].dynamic = container._properties[i];
     }
 
-    for (i = 0; i < size; i += this.size)
+    for (i = 0; i < size; i += batchSize)
     {
-        buffers.push( new ParticleBuffer(gl,  this.properties, this.size, this.shader) );
+        buffers.push( new ParticleBuffer(gl,  this.properties, batchSize, this.shader) );
```

Comment:
I believe a small mistake was made when resolving the conflict here, if you look at the current dev version ```this.size``` hasn't been replaced by ```batchSize```